roles/ecs_node/vars/main.yaml

```diff
@@ -0,0 +1 @@
+az_length: "{{ ecs_availability_zones | length }}"
```

Comment:
It would be better to call it `az_count` :)

Reply:
done